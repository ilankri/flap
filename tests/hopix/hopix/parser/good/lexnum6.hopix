val x = 0xdeadbeef
