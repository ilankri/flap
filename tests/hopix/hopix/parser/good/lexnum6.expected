
val x = -559038737
